### ISSUE TITLE
Exception does not render correctly

### ISSUE THREAD

**Issue by patriksvensson** [spectreconsole/spectre.console#489]
**Information**
 - OS: Windows
 - Version: 0.40.0
 - Terminal: Windows Terminal

**Describe the bug**

```
Could not load file or assembly 'Microsoft.Management.Infrastructure, Version=1.0.0.0, Culture=neutral,
PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Management.Infrastructure, Version=1.0.0.0,
Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified.
File name: 'Microsoft.Management.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
   at Cupboard.Internal.WmiFacts.GetFacts(IRemainingArguments args)+MoveNext()
   at Cupboard.Internal.FactBuilder.Build(IRemainingArguments args) in /_/src/Cupboard/Internal/FactBuilder.cs:line 20
   at Cupboard.Internal.ExecutionEngine.Run(IRemainingArguments args, IStatusUpdater status, Boolean dryRun) in
/_/src/Cupboard/Internal/ExecutionEngine.cs:line 34
   at Cupboard.Internal.RunCommand.Run(CommandContext context, Settings settings) in
/_/src/Cupboard/Internal/Cli/RunCommand.cs:line 136
   at Cupboard.Internal.RunCommand.ExecuteAsync(CommandContext context, Settings settings) in
/_/src/Cupboard/Internal/Cli/RunCommand.cs:line 106
   at Spectre.Console.Cli.CommandExecutor.Execute(IConfiguration configuration, IEnumerable`1 args) in
/_/src/Spectre.Console/Cli/Internal/CommandExecutor.cs:line 84
   at Spectre.Console.Cli.CommandApp.RunAsync(IEnumerable`1 args) in /_/src/Spectre.Console/Cli/CommandApp.cs:line 87
   at Spectre.Console.Cli.CommandApp.Run(IEnumerable`1 args) in /_/src/Spectre.Console/Cli/CommandApp.cs:line 61
   at Spectre.Console.Cli.CommandApp`1.Run(IEnumerable`1 args) in /_/src/Spectre.Console/Cli/CommandAppOfT.cs:line 42
   at Cupboard.CupboardHost.Run(String[] args) in /_/src/Cupboard/CupboardHost.cs:line 30
```

**Screenshots**

![image](https://user-images.githubusercontent.com/357872/125960300-db271388-5843-441d-8b2e-1875d01f42a9.png)

**Comment by antoniovalentini** [comment]
Hey 👋 
I'm trying to reproduce the issue, but with no success. I've also tried to raise an exception in a `Cupboard` project.
Any suggestion?

### Information

- OS: Windows
- dotnet sdk: 5.0 / 6.0
- spectre.console: 0.41.0
- Terminal: Windows Terminal (PowerShell/Prompt)

Result:

![image](https://user-images.githubusercontent.com/4293609/132945265-779d8fc9-04ab-4897-a730-9eb045cc5237.png)

Code:
```C#
using System;
using System.Net.Http;
using Spectre.Console;

Console.WriteLine("Hello World!");

try
{
    new HttpClient().Send(new HttpRequestMessage(HttpMethod.Get, "notexisting.com"));
}
catch (Exception ex)
{
    AnsiConsole.WriteException(ex);
}

Console.WriteLine("Bye World!");
```

**Comment by Genteure** [comment]
There is a `\n` before the exception message string `File name:`
The following code reproduces this bug successfully:

```csharp
try
{
    throw new Exception("something with \n newline in message");
}
catch (Exception ex)
{
    Console.WriteLine(ex.ToString());
    Console.WriteLine("\n\n");
    AnsiConsole.WriteException(ex);
}
```

![image](https://user-images.githubusercontent.com/11240579/132945894-df2302e0-202b-42f9-965a-0f2c6dc4d408.png)

- OS: Windows
- Version: 0.40.0
- Terminal: Microsoft Visual Studio Debug Console

**Comment by antoniovalentini** [comment]
Hi @Genteure, thanks for the hint! I can now reproduce it.

![image](https://user-images.githubusercontent.com/4293609/132949419-5f723f66-39cd-4be5-b533-a34c3b9bf254.png)

But,
I cannot reproduce it anymore if I clone the repo and use the most recent code (`MinVer: Calculated version 0.41.1-preview.0.13`)

![image](https://user-images.githubusercontent.com/4293609/132949395-724967d2-a553-4b9c-9acb-5828b9db8243.png)

Is it possible that this got resolved along the way? 🤔

**Comment by Genteure** [comment]
Looks like it got fixed in #513 https://github.com/spectreconsole/spectre.console/commit/e0815930123f9fdfa55c141107c5b0d4e25560b3

**Comment by patriksvensson** [comment]
@Genteure You are correct. Closing this issue.
